test/source/test.ts

```diff
@@ -129,6 +129,8 @@ ava.default.after.always('evaluate Catch.reportErr errors', async t => {
     ].includes(e.message))
     // below for test "user4@standardsubdomainfes.test:8001 - PWD encrypted message with FES web portal - a send fails with gateway update error"
     .filter(e => !e.message.includes('Test error'))
+    // below for test "setup - invalid csrf token returns error on gmail login"
+    .filter(e => !e.message.includes('failed to log into google in newGoogleAcctAuthPromptThenAlertOrForward'))
```

Comment:
That's what I meant. I don't want this reported, therefore this should be removed.

Reply:
Even though, `AssertError` inherits from `UnreportableError`, above code is necessary because of this line.
It returns result as error and so javascript error is occurred
```ts
if (receivedState !== expectedState) {
  return { acctEmail, result: 'Error', error: `Wrong oauth CSRF token. Please try again.`, id_token: undefined };
}
```

Reply:
But it affects other code, as that error is thrown in many other places. We are not testing that, but there are other places using `Assert.` that will now newly be throwing reportable errors, which is not intended.

Reply:
Anyway, please help me remove these two lines in test.ts. Rest looks good.

Reply:
Yeah. already inherited `AssertError` from `UnreportableError`.
I think you misunderstood me.
Those above 2 lines are not needed because of AssertError not inherit from UnreportableError
It's needed because we check `crsftoken`(state) and return error if it doesn't match.

Reply:
This line
```ts
return { acctEmail, result: 'Error', error: `Wrong oauth CSRF token. Please try again.`, id_token: undefined };
```

Reply:
I see. I would not want that reported, either. It can sometimes happen maybe if user uses the back and forward button on the host page and then re-confirms the old oauth form, that kind of thing. Could this be selectively not reported?

Reply:
Done